models/edr/system/information.sql

```diff
@@ -0,0 +1,9 @@
+{{
```

Comment:
cosmetics - maybe call it elementary_versions so it's more explicit?

Reply:
I thought of something more generic in which we'll store information rather than solely the versions.
What do you think?